packages/junon-matchmaker/src/index.js

```diff
@@ -881,6 +881,12 @@ class MatchmakerServer {
             return
           }
         }
+        if(sectorData.name.length > 255) {
```

Comment:
what if user provided a blank sector name? i.e didnt type anything on input field, would it still work?

Reply:
Yes, all this is doing is limiting the name so it doesn't go over 255 characters (VARCHAR(255) in sql)
No further restrictions are necessary. If a blank name is provided, it will display sector [sector id], and in-game it will display the creator's username.